packages/math-input/src/components/keypad/__tests__/keypad-v2-mathquill.test.tsx

```diff
@@ -231,4 +247,45 @@ describe("Keypad v2 with MathQuill", () => {
 
         expect(mockMathInputCallback).toHaveBeenLastCalledWith("");
     });
+
+    // Keypad event tests
+    it("fires the keypad open event on open", () => {
+        // Arrange
+        const mockSendEvent = jest.fn();
+        render(
+            <V2KeypadWithMathquill
+                onChangeMathInput={() => {}}
+                keypadClosed={true}
+                sendEvent={(event) => mockSendEvent(event.type)}
```

Comment:
One other little nit.  Typically, we just pass the mock function directly to the prop for these types of "callback props".  This means we can also assert that the right `virtualKeypadVersion` attribute is passed in the payload.  

So your assertion would become something like this: 

```js
        // Assert
        expect(mockSendEvent).toHaveBeenLastCalledWith({
            type: "math-input:keypad-opened",
            payload: {virtualKeypadVersion: "MATH_INPUT_KEYPAD_V2"},
        });
```